<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGraphics</class>
 <widget class="QWidget" name="ConfigureGraphics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string>Graphics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>API Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="api_widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="horizontalSpacing">
            <number>6</number>
           </property>
           <item row="4" column="0">
            <widget class="QWidget" name="backend_widget" native="true">
             <layout class="QHBoxLayout" name="backend_layout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="backend_label">
                <property name="text">
                 <string>Shader Backend:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="backend"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QWidget" name="device_widget" native="true">
             <layout class="QHBoxLayout" name="device_layout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="device_label">
                <property name="text">
                 <string>Device:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="device"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QWidget" name="api_layout_2" native="true">
             <layout class="QHBoxLayout" name="api_layout">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="api_label">
                <property name="text">
                 <string>API:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="api">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string notr="true">OpenGL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string notr="true">Vulkan</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Graphics Settings</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QCheckBox" name="use_disk_shader_cache">
          <property name="text">
           <string>Use disk pipeline cache</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="use_asynchronous_gpu_emulation">
          <property name="text">
           <string>Use asynchronous GPU emulation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="accelerate_astc">
          <property name="text">
           <string>Accelerate ASTC texture decoding</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="vsync_mode_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="vsync_mode_label">
             <property name="text">
              <string>VSync Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="vsync_mode_combobox">
             <property name="toolTip">
              <string>FIFO (VSync) does not drop frames or exhibit tearing but is limited by the screen refresh rate.
FIFO Relaxed is similar to FIFO but allows tearing as it recovers from a slow down.
Mailbox can have lower latency than FIFO and does not tear but may drop frames.
Immediate (no synchronization) just presents whatever is available and can exhibit tearing.</string>
             </property>
             <property name="currentText">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="nvdec_emulation_widget" native="true">
          <layout class="QHBoxLayout" name="nvdec_emulation_layout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="nvdec_emulation_label">
             <property name="text">
              <string>NVDEC emulation:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="nvdec_emulation">
             <item>
              <property name="text">
               <string>No Video Output</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CPU Video Decoding</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>GPU Video Decoding (Default)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="fullscreen_mode_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_1">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="fullscreen_mode_label">
             <property name="text">
              <string>Fullscreen Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="fullscreen_mode_combobox">
             <item>
              <property name="text">
               <string>Borderless Windowed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Exclusive Fullscreen</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="aspect_ratio_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="ar_label">
             <property name="text">
              <string>Aspect Ratio:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="aspect_ratio_combobox">
             <item>
              <property name="text">
               <string>Default (16:9)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Force 4:3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Force 21:9</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Force 16:10</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Stretch to Window</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="resolution_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="resolution_label">
             <property name="text">
              <string>Resolution:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="resolution_combobox">
             <item>
              <property name="text">
               <string>0.5X  (360p/540p) [EXPERIMENTAL]</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>0.75X (540p/810p) [EXPERIMENTAL]</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1X (720p/1080p)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1.5X (1080p/1620p) [EXPERIMENTAL]</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2X (1440p/2160p)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3X (2160p/3240p)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4X (2880p/4320p)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5X (3600p/5400p)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>6X (4320p/6480p)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>7X (5040p/7560p)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>8X (5760p/8640p)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="scaling_filter_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="scaling_filter_label">
             <property name="text">
              <string>Window Adapting Filter:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="scaling_filter_combobox">
             <item>
              <property name="text">
               <string>Nearest Neighbor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bilinear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bicubic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Gaussian</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ScaleForce</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>AMD FidelityFX™️ Super Resolution</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="anti_aliasing_layout" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="anti_aliasing_label">
             <property name="text">
              <string>Anti-Aliasing Method:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="anti_aliasing_combobox">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>FXAA</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SMAA</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="fsr_sharpening_layout" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="fsr_sharpening_label_group">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QComboBox" name="fsr_sharpening_combobox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>Use global FSR Sharpness</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Set FSR Sharpness</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="fsr_sharpening_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>FSR Sharpness:</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fsr_slider_layout">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QSlider" name="fsr_sharpening_slider">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="sliderPosition">
                <number>25</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="fsr_sharpening_value">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>100%</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="bg_layout" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="bg_combobox">
             <property name="currentText">
              <string>Use global background color</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="maxVisibleItems">
              <number>10</number>
             </property>
             <item>
              <property name="text">
               <string>Use global background color</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Set background color:</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="bg_label">
             <property name="text">
              <string>Background Color:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="bg_button">
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
